src/pages/ImportData.js

```diff
@@ -63,15 +64,22 @@ class Importer extends Component {
     const wsAjax = Ajax().Workspaces.workspace(namespace, name)
 
     try {
-      await Utils.switchCase(format,
-        ['entitiesJson', () => wsAjax.importJSON(url)],
-        ['PFB', () => wsAjax.importPFB(url)],
-        [Utils.DEFAULT, () => wsAjax.importBagit(url)]
-      )
-      if (format === 'PFB') {
-        notify('info', 'Data import in progress.', { message: 'Data will show up incrementally as the job progresses.' })
-      } else {
-        notify('success', 'Data imported successfully.', { timeout: 3000 })
+      switch (format) {
```

Comment:
Again, imperative code so I changed this to a `switch`. Also, I needed `jobId` for the async import so I could dismiss the notification when the import completes.